packages/core/test/integration/model.test.js

```diff
@@ -1012,7 +1034,9 @@ describe(Support.getTestDialectTeaser('Model'), () => {
             if (dialectName === 'sqlite3' && sql.includes('TABLE_INFO')) {
               test++;
               expect(sql).to.not.contain('special');
-            } else if (['mysql', 'mssql', 'mariadb', 'db2', 'ibmi'].includes(dialectName)) {
+            } else if (
+              ['mysql', 'mssql', 'mariadb', 'db2', 'ibmi', 'oracle'].includes(dialectName)
+            ) {
```

Comment:
This is the default behaviour. You can assume that `snowflake` also supports this. So if I'm not mistaken we only need to disable this for `postgres` (and possibly `sqlite3` depending if there is a case that `sql` does not include `TABLE_INFO`)